lib/plugin-hmr.js

```diff
@@ -40,10 +40,12 @@ module.exports = function (options = { bundleId: '' }) {
                     var instanceIds = Object.keys(instances).map(k => parseInt(k));
                     var disposable = [];
                     var acceptable = [];
+                    var acceptable_branches = {};
```

Comment:
why not use `let` (and `const` where appropriate) instead of `var`?

Reply:
It's a fair question. This is code that's outputted directly to the bundle. It doesn't go through any transformation. So for compatibility with older browsers like Internet Explorer, code that's directly outputted is written using ES5. It's one of those situations where you can get compatibility essentially for free with minimal effort.

Reply:
makes sense. thanks for clarifying!